extensions/cache/deployment/pom.xml

```diff
@@ -21,7 +21,7 @@
         </dependency>
         <dependency>
             <groupId>io.quarkus</groupId>
-            <artifactId>quarkus-arc-deployment</artifactId>
+            <artifactId>quarkus-smallrye-context-propagation-deployment</artifactId>
```

Comment:
Can we keep the Arc dependency if it's needed by this extension?

Relying on transitive dependencies is not a good idea if your extension uses directly ArC.

Reply:
Fixed.